project/src/main/webapp/event-details.js

```diff
@@ -83,21 +83,37 @@ function getLinkForOpportunity(opportunityId) {
 }
 
 /**
- * Gets event details from database and fills out event page with details
+ * Gets event details from database with eventId and fills out event page with details
  */
-function getEventDetails() {
+async function getEventDetails() {
+    //make sign up link go to correct
```

Comment:
Indentation on this comment 
(and in general, space before comment starts to separate the // )